src/core/GameClass.java

```diff
@@ -0,0 +1,49 @@
+package core;
+
+
+import java.lang.String;
+import java.util.Scanner;
+
+
+
+public class GameClass {
+    static String command;
+    static public void main(String[]args) {
+        Scanner input = new Scanner(System.in);
+        System.out.println("hello!");
+        System.out.println("Welcome to my first game.");
+        System.out.println("This is my first experience in game developing.");
+        System.out.println("There are some commands, which help you to play.");
+        System.out.println("Write (/rules) to see all the commands.");
+
+        command = input.nextLine();
+        System.out.println(command);
+        switch(command) {
```

Comment:
Here.I decide to make switch block cause its better to check command here.After this i change default block for 1 method to do it less.